app/scripts/modules/core/src/reactShims/react.injector.ts

```diff
@@ -60,13 +62,15 @@ export class CoreReactInject extends ReactInject {
   }
 
   // Services
+  public get $http() { return this.$injector.get('$http'); }
```

Comment:
Are you using this in an upcoming PR? Any reason not to use `API`?

Reply:
I should move these to the Netflix pr. Can't use API because it's fetching user supplied urls

Reply:
I should move these to the Netflix pr. Can't use API because it's fetching user supplied urls

Reply:
Yeah, seeing that in the Netflix PR now... no harm in putting them in here, though.